packages/terra-button/src/Button.scss

```diff
@@ -33,7 +33,7 @@
     border-width: var(--terra-button-border-width, 1px);
     cursor: pointer;
     display: inline-block;
-    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-family: inherit;
```

Comment:
could probably flat out delete the property...

Reply:
Try that out in see. I know with textarea is was inheriting from system-ui, but if that doesn't happen with button, I'd remove this style.